typescript/infra/src/utils/safe.ts

```diff
@@ -0,0 +1,226 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainNameOrId,
+  MultiProvider,
+  getSafe,
+  getSafeService,
+} from '@hyperlane-xyz/sdk';
+import { Address, CallData } from '@hyperlane-xyz/utils';
+
+export async function getSafeAndService({
+  chain,
```

Comment:
I don't feel too strongly but noting that this param pattern where we have an object and destructure deviates from the pattern we have elsewhere. Slight preference to keep things consistent but will defer to you if you feel strongly this is better

Reply:
more of a style preference based on usage in the newer parts of the sdk/cli, but happy to defer to the current infra style